### ISSUE TITLE
centos package manager test failing

### ISSUE THREAD

**Issue by chavafg** [kata-containers/tests#1270]
```
13:58:23 [Serial Test] package manager update test check yum update 
13:58:23   should not fail
13:58:23   /tmp/jenkins/workspace/kata-containers-tests-fedora-vsocks-PR/go/src/github.com/kata-containers/tests/integration/docker/package_manager_test.go:89
13:58:23 Running command '/usr/bin/docker [docker run --cidfile /tmp/cid488236648/AvtmHsiEcVgz0O5ZNquPasXymB7rVi --runtime kata-runtime --rm -td --name AvtmHsiEcVgz0O5ZNquPasXymB7rVi centos sh]'
13:58:26 [docker run --cidfile /tmp/cid488236648/AvtmHsiEcVgz0O5ZNquPasXymB7rVi --runtime kata-runtime --rm -td --name AvtmHsiEcVgz0O5ZNquPasXymB7rVi centos sh]
13:58:26 Timeout: 60 seconds
13:58:26 Exit Code: 0
13:58:26 Stdout: bf2fdf520a631d5ed760a28042e0a09001a3ec89efce4d489ee1f394cbe0a501
13:58:26 
13:58:26 Stderr: 
13:58:26 Running command '/usr/bin/docker [docker exec AvtmHsiEcVgz0O5ZNquPasXymB7rVi yum -y update]'
13:59:26 .
14:00:26 .
14:01:26 .
14:02:26 .
14:03:26 .
14:04:26 .
14:05:26 .
14:06:26 .
14:07:26 .
14:08:26 .
14:09:26 .
14:10:26 .
14:11:26 .
14:12:26 .
14:13:26 Killing process timeout reached '900' seconds
14:13:26 Running command '/usr/bin/docker [docker exec AvtmHsiEcVgz0O5ZNquPasXymB7rVi yum -y update]'
14:14:26 .
14:15:26 .
14:16:26 .
14:17:26 .
14:18:26 .
14:19:26 .
14:20:26 .
14:21:26 .
14:22:26 .
14:23:26 .
14:24:26 .
14:25:26 .
14:26:26 .
14:27:26 .
14:28:26 Killing process timeout reached '900' seconds
14:28:26 Running command '/usr/bin/docker [docker exec AvtmHsiEcVgz0O5ZNquPasXymB7rVi yum -y update]'
14:29:26 .
14:30:26 .
14:31:26 .
14:32:26 .
14:33:26 .
14:34:26 .
14:35:26 .
14:36:26 .
14:37:26 .
14:38:26 .
14:39:26 .
14:40:26 .
14:41:26 .
14:42:26 .
14:43:26 Killing process timeout reached '900' seconds
14:43:26 Running command '/usr/bin/docker [docker exec AvtmHsiEcVgz0O5ZNquPasXymB7rVi yum -y update]'
14:44:26 .
14:45:26 .
14:46:26 .
14:47:26 .
14:48:26 .
```

**Comment by jcvenegas** [comment]
talking with @chavafg seems that the latest image is making this fail, seems like a network issue, and because this not happend in other distros makes me think in a race condition setting iptables or something?

**Comment by jcvenegas** [comment]
cached image removed and generated again as seen some jobs that build the image before were cached and worked. lets see if the image was corrupted

**Comment by chavafg** [comment]
when running this manually, the `yum update` gets stucked here:
```
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : tzdata-2018i-1.el7.noarch                                                                                                                                                                      1/28
  Updating   : glibc-2.17-260.el7_6.3.x86_64                                                                                                                                                                  2/28
```

**Comment by chavafg** [comment]
The issue happens when using image `kata-containers-clearlinux-28190-osbuilder-da9f541-agent-a2037c0.img.tar.xz`. I ran the `yum update` inside the centos container and at the same time I ran `top`. I see that the yum process uses 99% of the CPU and stays there while it is running . 

When downgrading to image `kata-containers-clearlinux-28100-osbuilder-da9f541-agent-a2037c0.img.tar.xz`, this issue does not happen. The `yum update` inside the container finishes correctly and the CPU usage of the container does not get above 60%.

So it seems that something between 28100 and 28190 in clearlinux changed that is making this test fail.
